crates/proc_macro_srv/src/abis/abi_1_56/rustc_server.rs

```diff
@@ -490,7 +490,7 @@ impl server::Literal for Rustc {
         "".to_owned()
     }
     fn from_str(&mut self, _s: &str) -> Result<Self::Literal, ()> {
-        unimplemented!()
+        Ok(Literal { text: _s.into(), id: tt::TokenId::unspecified() })
```

Comment:
`_s` is now used, so the underscore is unneeded

```suggestion
        Ok(Literal { text: s.into(), id: tt::TokenId::unspecified() })
```